Makefile

```diff
@@ -24,6 +24,8 @@ endif
 # - use environment variables to overwrite this value (e.g export DEFAULT_CHANNEL="stable")
 ifneq ($(origin DEFAULT_CHANNEL), undefined)
 BUNDLE_DEFAULT_CHANNEL := --default-channel=$(DEFAULT_CHANNEL)
+else
+DEFAULT_CHANNEL := alpha
```

Comment:
Should BUNDLE_DEFAULT_CHANNEL be set in the else case?

Reply:
Default is `alpha` and we do not need BUNDLE_DEFAULT_CHANNEL (retain as it exists)